### ISSUE TITLE
vscode not detecting the settings

### ISSUE THREAD

**Issue by themousepotato** [athityakumar/proxyrc#5]

**Comment by athityakumar** [comment]
Yup, VCode doesn't seem to inherit proxy from the env variables. We could add this as a workaround though: https://github.com/Microsoft/vscode/issues/46348#issuecomment-375368714
